src/main/java/org/ow2/proactive/procci/service/occi/MixinService.java

```diff
@@ -110,50 +110,25 @@ public Mixin getMixinByTitle(String title) throws ClientException {
      * @return a mixin without entities
      * @throws ClientException if there is an error in the cloud automation service response
      */
-    public Mixin getMixinMockByTitle(String title) throws ClientException {
+    public Mixin getMixinMockByTitle(String title) {
 
         MixinRendering mixinRendering = getMixinRenderingByTitle(title);
         return new MixinBuilder(this, instanceService, mixinRendering).buildMock();
     }
 
     /**
-     * Give a list of mixin from their titles
-     *
-     * @param titles is the mixins titles
-     * @return a list of Mixin
-     * @throws ClientException if there is an error in the cloud automation service response
-     */
-    public List<Mixin> getMixinsByTitles(List<String> titles) throws ClientException {
-        return titles.stream().map(title -> getMixinByTitle(title)).collect(Collectors.toList());
-    }
-
-    /**
-     * Give the list of the mixin name of an entity
-     *
-     * @param entityId is an occi entity
-     * @return the list of the mixin related to the entity instance
-     * @throws CloudAutomationServerException if there is an error in the cloud automation service response
-     */
-    public Set<String> getEntityMixinNames(String entityId) throws CloudAutomationServerException {
```

Comment:
This method was set as a package method and was moved in this class after the public method.